crates/kubelet/Cargo.toml

```diff
@@ -49,6 +49,7 @@ oci-distribution = { path = "../oci-distribution", version = "0.1.0" }
 url = "2.1"
 warp = { version = "0.2", features = ['tls'] }
 http = "0.2"
+config_file = { package = "config", version = "0.10.0" }
```

Comment:
We should probably turn off the default features and pull in only the formats we want to support so as to avoid any extra dependencies

Reply:
Cool; let's do that once we agree on the desired format(s).

Reply:
Removed in favour of plain ol' serde anyway